lemur/certificates/cli.py

```diff
@@ -775,3 +777,146 @@ def deactivate_entrust_certificates():
             current_app.logger.info(log_data)
             sentry.captureException()
             current_app.logger.exception(e)
+
+
+@manager.option("-c", "--commit", dest="commit", action="store_true", default=False, help="Persist changes.")
+def disable_rotation_of_duplicate_certificates(commit):
+    log_data = {
+        "function": f"{__name__}.{sys._getframe().f_code.co_name}",
+        "message": "Disabling auto-rotate for duplicate certificates"
+    }
+
+    if commit:
+        print("[!] Running in COMMIT mode.")
+
+    authority_names = current_app.config.get("AUTHORITY_TO_DISABLE_ROTATE_OF_DUPLICATE_CERTIFICATES")
+    if not authority_names:
+        log_data["message"] = "Skipping task: No authorities configured"
+        current_app.logger.debug(log_data)
+        return
+
+    log_data["authorities"] = authority_names
+
+    authority_ids = [a.id for a in Authority.query.filter(Authority.name.in_(authority_names)).all()]
```

Comment:
I'll add new method to `authorities/service.py` which takes multiple names - want to avoid separate DB call for each authority. I did consider using IDs in `AUTHORITY_TO_DISABLE_ROTATE_OF_DUPLICATE_CERTIFICATES`, but it gets less readable. We do not display authority ID on UI too, so needs a DB query to do the mapping.